templates/service.yaml

```diff
@@ -21,5 +21,11 @@ spec:
       {{- if and (eq (.Values.deploy.expose.type) "NodePort") (.Values.deploy.expose.nodePort) }}
       nodePort: {{ .Values.deploy.expose.nodePort }}
       {{- end}}
+    - port: 9999
```

Comment:
I think we want JMX to be optional, with it disabled by default.

I would suggest that we should add a `deploy.jmx.enabled` bool to the `values.yaml` that allows the user to dictate whether this port is exposed.